docs/misc/config.md

```diff
@@ -158,9 +159,35 @@ Operations that occur whenever a config is loaded or reloaded can be done using
 :::caution
 These events are called for all configurations for the mod; the `ModConfig` object provided should be used to denote which configuration is being loaded or reloaded.
 :::
+## Configuration Screen
+
+A mod can use the build-in configuration screen that NeoForge provides. A configuration screen allows users to edit the config values for a mod while in-game without needing to open any files. The screen will automatically parse your registered config files and populate the screen. Mods can extend ConfigurationScreen to change the behavior of the default screen or make their own configuration screen.
+
+A configuration screen can be registered for a mod by registering a `IConfigScreenFactory` extension point during mod construction on the [client][client]:
+```java
+// In the main client mod file
+public ExampleModClient(ModContainer container) {
+    // This will use NeoForge's ConfigurationScreen to display this mod's configs
+    container.registerExtensionPoint(IConfigScreenFactory.class, ConfigurationScreen::new);
+}
+```
+
+The configuration screen can be accessed in game by going to the 'Mods' page, selecting the mod from the sidebar, and clicking the 'Config' button. Startup, Common, and Client config options will always be editable at any point. Server configs are only editable in the screen when playing on a world locally. If connected to a server or to another person's LAN world, Server config option will be disabled in the screen.
+
+:::warning
+Be sure to add translations for all configs! After visiting all of the mod's config screen pages and then back out to the mod list, all untranslated config entries that were encountered will be printed to the console. This makes it easier to know what to translate and what the translation keys are. 
```

Comment:
This sentence feels a bit awkward. It needs to be more direct at saying, 'add translations. To help, anything without a translation key will be printed to the console.'

Reply:
cleaned up now